bash-completion/ipcrm

```diff
@@ -6,32 +6,47 @@ _ipcrm_module()
 	prev="${COMP_WORDS[COMP_CWORD-1]}"
 	case $prev in
 		'-m'|'--shmem-id')
-			KEYIDS="$(ipcs -m | awk '{if (3 < NR) {print $2}}')"
+			KEYIDS="$(lsipc -m | awk 'NR>1 {print $2}')"
```

Comment:
All ls-like tools in util-linux should be able to print just one value; it means awk should be unnecessary. Also note that default output (without -o) is subject to change.

`KEYIDS=$(lsipc -m --noheadings -o ID)`